docs/content/integrations/airflow.mdx

```diff
@@ -123,6 +123,35 @@ The `image` argument is the name of the Docker image. Running in a containerized
 
 If you want your containerized job to be available to Airflow operators running on other machines (for example, in environments where Airflow workers are running remotely) you'll need to push your Docker image to a Docker registry so that remote instances of Docker can pull the image by name, or otherwise ensure that the image is available on remote nodes.
 
+## Ingesting Airflow operators
+
+Dagster can convert Airflow operators to Dagster ops, for any Airflow operators that require no instance-wide configuration via `airflow.cfg`. Using <PyObject module="dagster_airflow" object="airflow_operator_to_op"/>, you can convert instantiated Airflow operators to be executed within a Dagster op. Optionally, you can pass in a list of Airflow connection objects utilized by the operator.
```

Comment:
```suggestion
Dagster can convert Airflow operators to Dagster ops for any Airflow operators that require no instance-wide configuration via `airflow.cfg`. Using <PyObject module="dagster_airflow" object="airflow_operator_to_op"/>, you can convert instantiated Airflow operators to be executed within a Dagster op. Optionally, you can pass in a list of Airflow connection objects utilized by the operator:
```

Small things - removing an unneeded comma and adding a colon to the last sentence to tie it to the example.